pkg/assetfetcher/http.go

```diff
@@ -175,3 +175,20 @@ func (c *NexposeAssetFetcher) newNexposeSiteAssetsRequest(siteID string, page in
 	req, _ := http.NewRequest(http.MethodGet, u.String(), http.NoBody)
 	return req
 }
+
+// CheckDependencies makes a call to the nexpose endppoint "/api/3/scans".
```

Comment:
Task: this comment line is now out of date. Please fix.

Reply:
gotcha